test/JDBC/input/storedProcedures/Test-sp_reset_connection.mix

```diff
@@ -54,3 +55,45 @@ GO
 GO
 select transaction_isolation_level from sys.dm_exec_sessions where session_id=@@SPID
 GO
+
+-- 6. Test Database Context being reset
+--      Tests include negative cases where db is dropped or renamed
+Create database reset_con_db1;
+GO
+Create database reset_con_db2;
+GO
+
+-- tsql database=reset_con_db1
+select db_name();
```

Comment:
Along with current db_name, Let's validate current_user and search_path since reset connection affects all of them.

Reply:
This is a diff task which is in progress, we are focusing on database being reset here